src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java

```diff
@@ -1,18 +1,39 @@
 package com.ccr4ft3r.geotaggedscreenshots.util;
 
+import com.ccr4ft3r.geotaggedscreenshots.ModConstants;
 import com.mojang.blaze3d.systems.RenderSystem;
-import com.mojang.blaze3d.vertex.PoseStack;
-import net.minecraft.client.gui.GuiComponent;
+import com.mojang.blaze3d.vertex.*;
+import net.minecraft.client.gui.GuiGraphics;
 import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.resources.ResourceLocation;
+import org.joml.Matrix4f;
 
 public class RenderUtil {
 
-    public static void renderImage(PoseStack matrixStack, int imageId, int width, int height, int scaledWidth, int scaledHeight, int xOffset, int yOffset) {
+    public static void renderImage(GuiGraphics guiGraphics, int imageId, int width, int height, int scaledWidth, int scaledHeight, int xOffset, int yOffset) {
         RenderSystem.setShader(GameRenderer::getPositionTexShader);
         RenderSystem.setShaderColor(1.0f, 1.0f, 1.0f, 1.0f);
         RenderSystem.setShaderTexture(0, imageId);
         RenderSystem.enableBlend();
-        GuiComponent.blit(matrixStack, xOffset, yOffset, scaledWidth, scaledHeight, 0, 0, width, height, width, height);
+        blit(guiGraphics, xOffset, yOffset, scaledWidth, scaledHeight, 0, 0, width, height, width, height);
         RenderSystem.disableBlend();
     }
+    public static void blit(GuiGraphics guiGraphics, int p_283671_, int p_282377_, int p_282058_, int p_281939_, float p_282285_, float p_283199_, int p_282186_, int p_282322_, int p_282481_, int p_281887_) {
+        blit(guiGraphics, p_283671_, p_283671_ + p_282058_, p_282377_, p_282377_ + p_281939_, 0, p_282186_, p_282322_, p_282285_, p_283199_, p_282481_, p_281887_);
+    }
+
+    static void blit(GuiGraphics guiGraphics, int p_282732_, int p_283541_, int p_281760_, int p_283298_, int p_283429_, int p_282193_, int p_281980_, float p_282660_, float p_281522_, int p_282315_, int p_281436_) {
+        innerBlit(guiGraphics, p_282732_, p_283541_, p_281760_, p_283298_, p_283429_, (p_282660_ + 0.0F) / (float)p_282315_, (p_282660_ + (float)p_282193_) / (float)p_282315_, (p_281522_ + 0.0F) / (float)p_281436_, (p_281522_ + (float)p_281980_) / (float)p_281436_);
```

Comment:
What's the intention of adding zero to int? `p_281522_ + 0.0F`
The second cast `(float)p_281436_` should be sufficient (for an int division that should result in a float only one operand needs to be of the type float)

Reply:
I just direct copied those things haha,gonna change it now